### PR TITLE
Warn if automatic updates is disabled in Kodi

### PR DESCRIPTION
**Pull request by dagwieers**
This will check whether automatic updates are enabled or not.

This relates to #764

### CODE REVIEW

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -511,8 +511,16 @@ msgid "Update add-on repositories"
...
 
 msgctxt "#30451"
-msgid "If a VRT NU add-on update is available, it will now be updated and you will be notified.\nMake sure you have [I]Auto-update[/I] enabled in Kodi and the VRT NU add-on."
-msgstr "Als een VRT NU update beschikbaar is, zal deze nu worden geïnstalleerd en krijg je hiervan een melding.\nControleer wel dat [I]Automatische update[/I] is ingesteld voor Kodi en de VRT NU add-on."
+msgid "If a VRT NU add-on update is available, it should now be updating and you will be notified.\nMake sure you have [I]Auto-update[/I] enabled for the VRT NU add-on."
+msgstr "Als een VRT NU update beschikbaar is, zal deze nu geïnstalleerd worden en krijg je hiervan een melding.\nControleer wel dat [I]Automatische update[/I] is ingesteld voor de VRT NU add-on."
+
+msgctxt "#30452"
+msgid "Automatic updates are disabled"
+msgstr "Automatische updates zijn niet geactiveerd"
```
Comment (mediaminister) [inline resources/language/resource.language.nl_nl/strings.po:519]: uitgeschakeld = niet geactiveerd

File: resources/language/resource.language.en_gb/strings.po [hunk 2]
```diff
@@ -788,9 +797,17 @@ msgctxt "#30891"
...
+msgid "Install the latest add-on updates…"
+msgstr "Installeer de recentste add-on updates…"
+
+msgctxt "#30895"
+msgid "Open Kodi add-ons settings… [COLOR=gray]to enable Kodi auto-updates)[/COLOR]"
+msgstr "Open de Kodi add-ons instellingen… [COLOR=gray](voor Kodi automatische updates)[/COLOR]"
+
+msgctxt "#30897"
+msgid "Open VRT NU add-on information…"
+msgstr "Open VRT NU add-on informatie… [COLOR=gray](voor VRT NU automatisch updates)[/COLOR]"
```
Comment (mediaminister) [inline resources/language/resource.language.en_gb/strings.po:810]: You added Dutch translations in resource.language.en_gb/strings.po

File: resources/language/resource.language.nl_nl/strings.po [hunk 3]
```diff
@@ -511,8 +511,16 @@ msgid "Update add-on repositories"
...
+
+msgctxt "#30453"
+msgid "Kodi is configured to not update add-ons automatically, so you will not get VRT NU updates unless you fix this setting.\nWe will open de Kodi settings, please change this setting."
+msgstr "Kodi is ingesteld om niet automatisch add-ons bij te werken, dus je zal geen VRT NU updates ontvangen tenzij je deze instelling aanpast.\nWe openen nu de Kodi settings zodat je dit kan wijzigen."
```
Comment (mediaminister) [inline resources/language/resource.language.nl_nl/strings.po:523]: Andere woordvolgorde: Kodi is ingesteld om  add-ons niet automatisch bij te werken

Comment (mediaminister) [reply resources/language/resource.language.nl_nl/strings.po:523]: Kodi settings = Kodi-instellingen

File: resources/settings.xml [hunk 4]
```diff
@@ -71,7 +71,9 @@
     </category>
     <category label="30890"> <!-- Updates -->
         <setting label="30891" type="lsep"/> <!-- Update the VRT NU add-on -->
-        <setting label="30892" help="30893" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/update/repos)" option="close"/>
+        <setting label="30893" help="30894" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/update/repos)" option="close"/>
+        <setting label="30895" help="30896" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/show/settings/addons)" option="close"/>
+        <!-- setting label="30897" help="30898" type="action" action="RunPlugin(plugin://plugin.video.vrt.nu/show/information" option="close"/ -->
```
Comment (mediaminister) [inline resources/settings.xml:76]: Is this functional?

File: resources/language/resource.language.en_gb/strings.po [hunk 5]
```diff
@@ -788,9 +797,17 @@ msgctxt "#30891"
 msgid "Update the VRT NU add-on"
 msgstr ""
 
-msgctxt "#30892"
-msgid "Update Kodi repositories and install add-on updates…"
-msgstr ""
+msgctxt "#30893"
+msgid "Install the latest add-on updates…"
+msgstr "Installeer de recentste add-on updates…"
```
[the remark this answers is not in the document]
Comment (dagwieers) [reply resources/language/resource.language.en_gb/strings.po:802]: That's a first! :laughing:

File: resources/language/resource.language.nl_nl/strings.po (hunk 1 above)
Comment (mediaminister) [reply resources/language/resource.language.nl_nl/strings.po:519]: Make sure = Zorg ervoor dat...

File: resources/lib/addon.py [hunk 6]
```diff
@@ -333,8 +333,24 @@ def iptv_epg():
...
+@plugin.route('/show/settings/addons')
+def show_settings_addons():
+    """Open the Kodi System Settings in the Add-ons category"""
+    execute_builtin('ActivateWindow(SystemSettings,addons)')
+
+
+@plugin.route('/open/information')
+def show_information():
+    """Show the Add-on information pane"""
+    execute_builtin('ActivateWindow(SystemSettings,addons)')
```
Comment (dagwieers) [inline resources/lib/addon.py:353]: I left this in for someone to try, but since it is not functional we might as wel remove it.

**Comment by dagwieers** [comment]
So this is the best we can do IMO, even though I prefer not to have to add this...

@michaelarnauts It probably helps to add this to VTM GO and VIER-VIJF-ZES too.

**Pull request merged**